cypress-io/cypress
# fix(svelte): default mount log to true

### User facing changelog
Logs the Svelte mount command by default

### Additional details
The mount log should default to true, but we missed the fallback. Found when pairing with @rockindahizzy (thanks Rocky!)

### Steps to test
Throw a `.only` on [svelte component testing](https://github.com/cypress-io/cypress/blob/b902297935b13ad11e763cef6bc02ab4c649e7c4/system-tests/test/component_testing_spec.ts#L136) and run `yarn workspace @tooling/system-tests test component_testing_spec.ts --no-exit --headed`, or test locally against a Svelte app. Check the reporter for a `mount <MyComponent ...>` log.

### How has the user experience changed?
Before:
![Screen Shot 2022-09-09 at 4 06 10 PM](https://user-images.githubusercontent.com/25158820/189445863-77d04d2e-9da9-4070-904d-a172ed9e9ded.png)

After:
![Screen Shot 2022-09-09 at 4 05 10 PM](https://user-images.githubusercontent.com/25158820/189445879-6097274d-568b-48ab-a188-38cdd195adf9.png)

### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
